Formula/openmittsu.rb

```diff
@@ -0,0 +1,33 @@
+class Openmittsu < Formula
+  desc "Cross-platform GPLv2 desktop client for the Threema Messenger App"
+  homepage "https://github.com/blizzard4591/openMittsu"
+  url "https://github.com/blizzard4591/openMittsu/archive/0.9.15.tar.gz"
+  sha256 "7c70de775716f8c01c6f6f4c812fd938f2aea979f6fe4562bb6a4089fa3b9cae"
+  head "https://github.com/blizzard4591/openMittsu.git", :using => :git, :shallow => false
+
+  depends_on "blizzard4591/homebrew-qt5-sqlcipher/qt5-sqlcipher"
+  depends_on "cmake"
+  depends_on "libsodium"
+  depends_on :macos => :mavericks
```

Comment:
Does it only work with mavericks?

Reply:
No, I got this from a formula that I used as reference, but did not really know what this does. Will remove.
But shouldnt this mean "any version greater or equal mavericks"?

Reply:
> any version greater or equal mavericks.

That's implied by being in homebrew since we only support the last 3 versions. Since it's cross-platform it shouldn't depend on a macOS version iyam